aiida/restapi/translator/base.py

```diff
@@ -399,7 +323,29 @@ def set_query(
 
         ## Add projections
         if projections is None:
-            self.set_default_projections()
+            if attributes is None and extras is None:
+                self.set_default_projections()
+            else:
+                default_projections = self.get_default_projections()
+                if attributes in [True, 'true', 'True']:
```

Comment:
As discussed, I wouldn't use this logic, because if you want only the attribute called `true` you cannot do it

Reply:
Changed to return also attributes called `true` or `false`.

Reply:
Let's be type-sensitive, only accept `True` or `False`, and return 400 for other values

Reply:
This is fixed, only `true` is accepted (lower-case and not within double-quotes) all other values return 400